assets/src/dashboard/components/popoverMenu/popoverCard.js

```diff
@@ -252,7 +253,8 @@ function PopoverCard({ children, isOpen }) {
   return (
     <ButtonInner
       ref={menuTogglePositionRef}
-      isOpen={isOpen && isReady}
+      isOpen={isOpen}
```

Comment:
doesn't seem like this is used anymore in `ButtonInner` should it be removed?

Reply:
Since `ButtonInner` is build off of `Popover` we have to keep it - Popover is in charge of actually showing the menu.